src/components/application_manager/src/display_capabilities_builder.cc

```diff
@@ -102,6 +105,11 @@ DisplayCapabilitiesBuilder::display_capabilities() const {
   return display_capabilities_;
 }
 
+bool DisplayCapabilitiesBuilder::is_window_resumption_needed() const {
+  LOG4CXX_AUTO_TRACE(logger_);
```

Comment:
@ychernysheva trace is redundant in getters

Reply:
Fixed in 041b18b68fcb8cae8c7116d2442b340553734285